fair-ca.po

```diff
@@ -0,0 +1,206 @@
+# Copyright (C) 2025 FAIR Contributors
+# This file is distributed under the GPLv2.
+msgid ""
+msgstr ""
+"Project-Id-Version: FAIR - Federated and Independent Repositories 0.1\n"
+"Report-Msgid-Bugs-To: https://github.com/fairpm/fair-plugin/issues\n"
+"Last-Translator: Marc Armengou\n"
+"Language-Team: Català\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"POT-Creation-Date: 2025-06-10T04:28:22+00:00\n"
+"PO-Revision-Date: 2025-06-18 13:44+0000\n"
+"X-Generator: Loco https://localise.biz/\n"
+"X-Domain: fair\n"
+"Language: ca\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Loco-Version: 2.8.0; wp-6.8.1; php-8.2.27"
+
+#. translators: %s: The version string.
+#: inc/credits/namespace.php:85
+#, php-format
+msgid "%s is not a valid version string."
+msgstr "%s no és una cadena de versió vàlida."
+
+#. translators: %s: theme name
+#: inc/updater/class-lite.php:407
+#, php-format
+msgid "%s: update now"
+msgstr "%s: actualitza ara"
+
+#: inc/avatars/namespace.php:256
+msgctxt "alt for unknown avatar user"
+msgid "profile picture for user"
+msgstr "imatge de perfil per a l'usuari"
+
+#: inc/updater/class-lite.php:419
+#| msgid "%1$sAutomatic update is unavailable for this theme.%2$s"
+msgid "Automatic update is unavailable for this theme."
+msgstr "L'actualització automàtica no està disponible per a aquest tema."
+
+#: inc/settings/namespace.php:107
+msgid "Avatar Settings"
+msgstr "Configuració de l'avatar"
+
+#: inc/settings/namespace.php:112
+msgid "Avatar Source"
+msgstr "Font de l'avatar"
+
+#: inc/settings/namespace.php:124
+msgid "Avatars will be loaded from the selected source."
+msgstr "Es pujaran els avatars des de la font seleccionada."
+
+#: inc/version-check/namespace.php:101
+msgctxt "browser version check"
+msgid "your browser"
+msgstr "el vostre navegador"
+
+#: inc/version-check/namespace.php:100
+msgctxt "browser version check"
+msgid "your platform"
+msgstr "la vostra plataforma"
+
+#: inc/avatars/namespace.php:93
+msgid "Choose Profile Image"
+msgstr "Trieu una imatge de perfil"
+
+#: assets/js/fair-avatars.js:20
+msgid "Choose Profile Picture"
+msgstr "Trieu la imatge del perfil"
+
+#: inc/dashboard-widgets/namespace.php:101
+msgctxt "default event location"
+msgid "Online"
+msgstr "En línia"
+
+#: inc/dashboard-widgets/namespace.php:186
+msgid "Events (by The WP World)"
+msgstr "Esdeveniments (pel WP World)"
```

Comment:
Technically, the name of the organization here is "The WP World". I'm not sure what the standard is for translating articles when they're part of the organization name.